shared/auth.py

```diff
@@ -133,6 +133,17 @@ async def failure(self, msg, exc=None, title='Failure'):
 -----END {blockchain} SIGNED MESSAGE-----
 '''
 
+def verify_recover_message_digest(sig, digest):
```

Comment:
`verify_recover_pubkey` ?? --> maybe this method could also return address format (AF_<what>) based on parsed header byte?